config/eventing-istio.yaml

```diff
@@ -7,13 +7,14 @@ config:
         version: "4.12"
     release-v1.10:
       openShiftVersions:
-      - version: "4.14"
-      - onDemand: true
```

Comment:
was this intentional?

Reply:
Yeah, sorry. Updated the PR description with "Remove OCP 4.11 completely as it's not supported anymore". It was intentional.